src/schemata/definitions/info.yml

```diff
@@ -1,33 +1,35 @@
 ---
 type: object
-title: Info Section Schema
+title: Info
 description: |
-  The info section of a workflow or step template holds general information
-  about the template.
+  The info section of a Workflow (Step) Template holds general information about the template.
 required:
   - title
+  - uuid
+  - version
 properties:
   title:
     type: string
-    title: Title
+    maxLength: 200
     description: |
-      A title that will be used as a display name to the workflow template.
+      The display name of the template in Laboperator.
     examples:
-      - My Premium Workflow
-
+      - PTC Integration
+  uuid:
+    $ref: '#/definitions/uuid'
+  version:
+    $ref: '#/definitions/version'
   description:
     type: string
-    title: Description
-    default: ''
+    maxLength: 10000
```

Comment:
This field was added.

<details>
<summary>10000 chars</summary>

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic. Quos repellendus sapiente accusamus quaerat, cumque facere perferendis quo iusto magnam omnis eveniet! Est sapiente laudantium quia, cum consectetur quo reiciendis assumenda, sunt, inventore quibusdam iusto dolores. Veritatis dolores blanditiis iusto omnis fugit cum magnam consectetur perspiciatis dolorem corporis sint sit ex esse nostrum, deserunt dolor delectus a voluptates amet eaque. Reprehenderit nam nemo esse provident accusamus nulla fuga dolorum aspernatur voluptatem, earum, explicabo repudiandae pariatur veniam, consequuntur a qui?

Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro libero inventore voluptatibus ullam temporibus possimus ad odio fuga tempora atque qui consectetur eum voluptates numquam, commodi repellendus, architecto dolorum officiis nihil exercitationem, quidem impedit. Accusantium iusto vel placeat! Nihil ipsa aliquam maiores minus eum quos aperiam, minima aut nulla ipsam cumque, ex impedit assumenda velit ab tempore distinctio. Magnam repellendus doloribus suscipit itaque. Assumenda officia sed blanditiis modi iure quos voluptatibus labore? Dolorem nisi obcaecati adipisci officia eos porro laboriosam beatae ipsum molestiae eligendi recusandae libero sunt expedita veritatis dicta saepe vel, aliquam esse dolores voluptate, rerum, in enim mollitia deserunt? Ipsa omnis magni in eligendi vel facilis soluta vero officiis reprehenderit dolor eaque, cumque explicabo quo aliquam voluptatem distinctio vitae sit nam optio maxime nulla facere iste commodi hic.

</details>

Reply:
how did you came up with all these numbers? did you check existing templates?

Reply:
Suggestion based on what I felt is reasonable, plus some margin.

- step description serves no real purpose anyway, so 1000 should be more than enough.
- step titles should be much shorter than 200 chars, but I figured we don't want to limit users to much (despite the UI actually doing that)
- root descriptions is the only one that is maybe debatable, but 10.000 is a pretty long wall of text. Nothing in our workflow template repo come even anywhere near this